pkg/acsengine/engine.go

```diff
@@ -487,12 +487,16 @@ func getParameters(cs *api.ContainerService, isClassicMode bool) (map[string]int
 	return parametersMap, nil
 }
 
+func AddValue(m map[string]interface{}, k string, v interface{}) { addValue(m, k, v) }
```

Comment:
I didn't want to make changes to existing code, so this allowed me to call the exported function without having to change existing code. See note above, happy to change just explaining my reasoning.